server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -488,6 +488,9 @@ public class IoTDBConfig {
 
   private boolean enableCompactionValidation = true;
 
+  /** The number of candidate compaction task queue. */
+  private int candidateCompactionTaskQueueSize = 50;
```

Comment:
This param is appeared in IoTDBConfig but not in iotdb-common.properties.

Reply:
Resolved.